js/app/vite.config.ts

```diff
@@ -147,7 +155,12 @@ export default defineConfig(({ mode }) => {
 				inspector: true,
 				compilerOptions: {
 					dev: true,
-					discloseVersion: false
+					discloseVersion: false,
+					accessors: mode === "test"
```

Comment:
I've turned on `accessors` for all components during tests, so we can just do `component.value` to get the value and `component.value = ...` to set the value. Easier than messing around with component methods. It adds extra code so we don't want to enable it all of the time.